README.md

```diff
@@ -27,11 +27,11 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
 * How to configure zip deployment:
   * If you set the deployment source of your web app to “None” (the default source on app creation), the deploy command will Zip the contents of a selected folder and upload the Zip file to Azure.
 
-  * VS Code will prompt on deploy if you would like to configure your project for faster deployment.  If you click "Yes", your project's settings will be changed in the following:
-    * 'appService.zipIgnorePattern' will include build artifacts to be ignored for zipping.  These will now be built on the server by running the appropriate build command.
-    * A '.deployment' file will be created in the root of the project.  This file configures 'SCM_DO_BUILD_DURING_DEPLOYMENT=true,' enabling build on deploy.
+  * VS Code will prompt on deploy if you would like to configure your project for faster deployment.  If you click "Yes", the following changes will be made in your project:
+    * The vscode setting 'appService.zipIgnorePattern' is changed to include build artifacts to be ignored for zipping.  These will be built on the server by running the appropriate build command.
```

Comment:
include build artifacts to be ignored is confusing  suggest:

is changed to indicate build artifacts that will not be zipped and deployed.